### PR TITLE
Fixes #27117 - Update scoped search descriptions

### PR DESCRIPTION
**Pull request by ofedoren**
Updates API docs for some of resources to contain some reference for users trying to use scoped search in api.

### CODE REVIEW

File: app/controllers/api/v2/ansible_variables_controller.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ def show; end
 
       api :GET, '/ansible_variables', N_('List Ansible variables')
       param_group :search_and_pagination, ::Api::V2::BaseController
+      add_scoped_search_description_for(AnsibleRole)
```
Comment (xprazak2) [inline app/controllers/api/v2/ansible_variables_controller.rb]: Should this be `AnsibleVariable`?

Comment (ofedoren) [reply app/controllers/api/v2/ansible_variables_controller.rb]: Sure it should :) thanks.

**Comment by ofedoren** [comment]
@xprazak2, updated.

**Comment by xprazak2** [comment]
This probably has no impact on Renderer test.

**Comment by ares** [comment]
Thanks @ofedoren, merging!

**Pull request merged**